titan/params/haart.yml

```diff
@@ -0,0 +1,22 @@
+haart:
+  use_cap:
+    default: false
+    description: Whether "prob" defined in demographics.race.sex_type.haart is a percent of diagnosed, HIV+ agents instead of a per-time-step probability for HAART coverage
```

Comment:
should this be `cap`?

Reply:
Making this and the reinit option just `use_[thing]` unless you have a better idea. just `cap` would make me think it's a value

Reply:
I meant in the description, I think the param this is related to is now `cap`, not `"prob"`

Reply:
ohhhhhhhhhhh